modules/Discord Integration/includes/endpoints/SyncDiscordRolesEndpoint.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @param int $user The NamelessMC user ID to edit
+ * @param string $roles An array of Discord Role ID to give to the user
+ *
+ * @return string JSON Array
+ */
```

Comment:
```suggestion
```

Reply:
It is a bit strange, but other endpoints have this documentation too. In any case, I forgot to update it so it has to go.